src/main/scala/dxWDL/dx/DxJob.scala

```diff
@@ -14,8 +14,7 @@ case class DxJobDescribe(project: String,
                          applet: DxApplet,
                          executableName: String,
                          parentJob: Option[DxJob],
-                         analysis: Option[DxAnalysis],
-                         output: Option[JsValue])
+                         analysis: Option[DxAnalysis])
```

Comment:
Not sure why we don't need output from the job anymore.

Reply:
It is never used on the describe object. The only place the output field is used is in WorkflowOutputReorg, and it is parsed directly out of the JSON there.